### ISSUE TITLE
Веб-доступ к Windows-клиенту

### ISSUE THREAD

**Issue by NoAdO** [DEgITx/rats-search#44]
Видел инструкцию по линуксовой версии. У виндового клиента есть возможность показывать морду через браузер? Если клиент будет запущен на своей файлопомойке, должно быть актуально.

**Comment by DEgITx** [comment]
Там инструкция под винду ровным счетом такаяже, если по ней пройтись то и на винде запуститься - веб-версия сейчас запускается отдельно

**Comment by NoAdO** [comment]
Значит я где-то не там читал. Там где я читал было "скопируйте, запустите какие-то команды, готово". Сейчас буду искать тогда.

**Comment by NoAdO** [comment]
Так и не нашёл ничего по серверной морде на винды. Краткое руководство для линукса, особенности на линуксе. В json файле есть http порт, но больше ничего. Видимо надо будет написать что-то типа краткого руководства(

**Comment by DEgITx** [comment]
в виде принцип такой же, скачивается репозиторий, устанавливается node, и выполняется последовательность команд из документации - веб морда доступна
просто интересно что подразумевалось - если прямо из клиента открыть как-то веб морду - то такое не предусмотрено, они шли особняком

**Comment by NoAdO** [comment]
Да, думал есть собранная версия или вариант установки как сервис по типу
resilio sync

вт, 17 июл. 2018 г., 8:52 Alexey Kasyanchuk <notifications@github.com>:

**Comment by Ruslan567vg** [comment]
на винде выдает чтото не скачалось 
PS C:\Users\ruslan> npm -v
6.5.0
PS C:\Users\ruslan> node -v
v11.8.0
PS C:\Users\ruslan> npm install
npm WARN saveError ENOENT: no such file or directory, open 'C:\Users\ruslan\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\ruslan\package.json'

**Comment by SpcCw** [comment]
> C:\Users\ruslan> npm install

Запускать надо из папки, куда скачан код rats-search из репозитория.
